src/Services/PopoverManager.vala

```diff
@@ -238,4 +253,18 @@ public class Wingpanel.Services.PopoverManager : Object {
             }
         });
     }
+
+    private void offset_notifications(Wingpanel.Widgets.IndicatorPopover? popover = null) {
+        int offset = 0;
+        if (popover != null) {
+            int minimum_height, natural_height;
+            popover.get_preferred_height (out minimum_height, out natural_height);
```

Comment:
You can use null instead of `out minimum_height` to get rid of unused variable.

Reply:
I cleaned up the code for this function, including the added newline.